src/toil/jobStores/aws/jobStore.py

```diff
@@ -299,6 +303,65 @@ def deleteFile(self, jobStoreFileID):
         else:
             info.delete()
 
+    def importFile(self, sourceUrl):
+        url = urlparse.urlparse(sourceUrl)
+        if url.scheme.lower() == 's3':
+            if len(sourceUrl.split('/')) != 4:
+                raise RuntimeError("The url '%s' is not a valid s3 url" % sourceUrl)
+            srcBucket = self._loadBucketOrFail(url.netloc)
```

Comment:
Consider refactoring url.netloc and url.path[1:] into more informative variable names like bucketName and keyName